src/mainwindow.cpp

```diff
@@ -0,0 +1,54 @@
+#include "mainwindow.h"
```

Comment:
I think this file should be merged into Atelier this way atilier owns the UI file. We can do things like change test or read stuff from the ui . instead of having a layer of extra signals and slots.